research/2022-12-05-Autofill_feature.md

```diff
@@ -0,0 +1,25 @@
+# Autofill Feature Usability Testing
+
+## Key Headlines
+
+- The question type page is visually easy for users to engage with but for some it is just on the edge of being too long and some believe it is important to be able to see all the questions on one page without scrolling down.
+- One user believed it would be easier if questions were ‘grouped’ into ‘chunks’ to reflect the beginning, middle and end of a form or by commonly used and custom questions. This raises the possibility of categorising question types and hints at the possibility users might benefit from ordering questions in a different way.
+- The ‘add a person’s name’ question was completed with ease and users welcomed the ability to break the name down into component parts. 
+- Users found it easy to add address questions and welcomed the addition of international and UK address fields.
+- Question settings are seen as a useful addition enabling users to see a summary of their questions and make changes easily. The language and layout is simple and straightforward to understand.
+- Optional questions were seen, understood and implemented by users with ease. 
+- The guidance and examples are working well with evidence of users being prompted to think about the questions they are creating, leading to better question design. They are also copy and pasting examples into the field which saves them time.
+- There were also some areas where users had some difficulty. Onto e question overview page, although the options are mostly understood there can be some problems understanding the language. For example, ‘selection from a list,’  which can be understood as a list of previous questions and ‘adding a number’ with some users unsure whether this would enable them to include other characters and symbols e.g. BH678 
+- Users had difficulty adding a ‘selection from a list’ because they expected the question page to be presented before the options. Currently the design is backwards. Some users indicated it would be even more beneficial to see the question and options on one page. 
```

Comment:
```suggestion
- Users had difficulty adding a ‘selection from a list’ because they expected the question page to be presented before creating the list of options. Currently the design is backwards. Some users indicated it would be even more beneficial to see the question and options on one page. 
```